ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,19 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- floor_position_mode:
```

Comment:
ah okay. I'm not the designer and on the surface probably not much benefit but I imagine it would be useful information to know the elevator usage trends when performing data analysis on the building information down the line?

Reply:
@kevin-hereworks is there an analog point available to provide floor information or only multi-state?